### PR TITLE
Show GPG messages about key expiration

### PR DESCRIPTION
**Pull request by dbnicholson**
Include the key expiration timestamp in the GPG verification result and show it in the description output. While here, also show if the key has been revoked.

I started working on a little more useful testing that involves creating GPG keys on the fly and changing their expiration times as needed, but I didn't get that done. I'm planning to do that for a related feature, but I figured I'd get this work out now instead of sitting on it. Let me know what you think.

### CODE REVIEW

**Comment by dbnicholson** [comment]
I pushed 2 more commits to do more detailed testing of key states by generating and manipulating keys on the fly with `gpg`. It works here but as you likely know `gpg` can be pretty fragile.

File: src/libostree/ostree-gpg-verify-result.c [hunk 1]
```diff
@@ -513,6 +513,49 @@ ostree_gpg_verify_result_describe (OstreeGpgVerifyResult *result,
...
 
+static void
+append_expire_info (GString *output_buffer,
+                    const gchar *line_prefix,
+                    const gchar *exp_type,
+                    gint64 exp_timestamp,
+                    gboolean expired)
+{
+  g_autoptr(GDateTime) date_time_utc = NULL;
+  g_autoptr(GDateTime) date_time_local = NULL;
```
Comment (cgwalters) [inline src/libostree/ostree-gpg-verify-result.c]: General preference in ostree is for declare-and-init, i.e. C99 not C89.

**Comment by cgwalters** [comment]
Not worth a respin just for that, overall looks great!  Thanks a lot!

@rh-atomic-bot r+ d8e2dfc

File: src/libostree/ostree-gpg-verify-result.c (hunk 1 above)
Comment (dbnicholson) [reply src/libostree/ostree-gpg-verify-result.c]: Copy and pasting there. I'll fix that for the next thing.

**Pull request closed without merging**